decredplugin/decredplugin.go

```diff
@@ -415,6 +416,55 @@ func DecodeVoteSummaryReply(payload []byte) (*VoteSummaryReply, error) {
 	return &v, nil
 }
 
+// BatchVoteSummary requests a summary of a set of proposal votes. This
+// includes certain voting period parameters and a summary of the vote
+// results.
+type BatchVoteSummary struct {
+	Tokens []string `json:"token"` // Censorship token
+}
+
+// EncodeBatchVoteSummary encodes BatchVoteSummary into a JSON byte slice.
+func EncodeBatchVoteSummary(v BatchVoteSummary) ([]byte, error) {
+	return json.Marshal(v)
+}
+
+// DecodeBatchVoteSummary decodes a JSON byte slice into a BatchVoteSummary.
+func DecodeBatchVoteSummary(payload []byte) (*BatchVoteSummary, error) {
+	var bv BatchVoteSummary
+
+	err := json.Unmarshal(payload, &bv)
+	if err != nil {
+		return nil, err
+	}
+
+	return &bv, nil
+}
+
+// BatchVoteSummaryReply is the reply to the VoteSummary command and returns certain
```

Comment:
80 column limit.

Are results always returned for every token that is passed in?  If not, this needs to be documented here.